### PR TITLE
Changes for PR #2

### PR DESCRIPTION
**Pull request by above9Sq**
This PR is for changes to PR #2

### CODE REVIEW

File: src/_includes/partials/contact/contact-us-box.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="[ box ] [ box__equal-space ]">
```
Comment (RainbowArray) [inline src/_includes/partials/contact/contact-us-box.html]: `box__equal-space` to `box--equal-space`

For the BEM style (Block-Element-Modifier just to be clear) of class names, we use `__` for child elements and `--` for modifications to the same element. We're doing the second one here.

File: src/scss/blocks/_box.scss [hunk 2]
```diff
@@ -8,6 +8,10 @@
   margin-bottom: get-size('600');
 }
 
+.box__equal-space {
```
Comment (RainbowArray) [inline src/scss/blocks/_box.scss]: box__equal-space to box--equal-space

**Review by RainbowArray — CHANGES_REQUESTED** [review]

**Comment by above9Sq** [comment]
All done. You've got the eagle eye!

**Comment by RainbowArray** [comment]
Looks great with changes!

**Pull request merged**